person.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'nameable'
 
 class Person < Nameable
+  has_many :rentals
```

Comment:
- [ ] Kindly add a method for adding rentals should be present. you can check this lesson  [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship),  so to achieve the association here this method add rental it should be added here and in book class as well sp `The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association 💯✅✅🔥🚀